modAionImpl/src/org/aion/zero/impl/cli/Cli.java

```diff
@@ -505,6 +575,7 @@ private void printHelp() {
         // the command line output has some styling characters in addition to the actual string
         // making the use of a regular expression necessary here
         usage = usage.replaceFirst(" \\[[^ ]*<hostname> <ip>.*]", "]");
+        usage = usage.replaceFirst("(?<=<enabled> )([\\s\\S]*?)]+", "| <slow_import> <frequency>");
```

Comment:
This seems to italicise the descriptions of the parameters, which is not how the other argument descriptions are formatted.

Reply:
fixed !